tests/__init__.py

```
@@ -0,0 +1,12 @@
+import pytest
+import os
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_test_env():
+    """Set up test environment variables"""
+    os.environ["SMTP_USERNAME"] = "test@example.com"
+    os.environ["SMTP_PASSWORD"] = "testpassword"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "SMTP_PASSWORD"

(S105)